webapp/components/LogInAccordion.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { Typography, Button, Accordion } from '@material-ui/core';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import Alert from '@material-ui/lab/Alert';
+import GetAppIcon from '@material-ui/icons/GetApp';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+import Input from 'components/Input';
+import styles from 'components/Form.module.css';
+import { fetchProfile, fetchToken } from 'service/medlemService';
+
+type Props = {
+    updateForm: (name: string, mailFrom: string, bankAccount: string, token: string) => void;
+  };
+const LogInAccordion = ({updateForm}: Props): JSX.Element => {
+
+    const [phoneNumber, setPhoneNumber] = useState('+47');
+    const [password, setPassword] = useState('');
+    const [error, setError] = useState('');
+    const [isSuccess, setIsSuccess] = useState(false);
+    const [expanded, setExpanded] = useState(false);
+
+    const fetchUserData = async () => {
+        try {
+            const trimmedPhoneNumber = phoneNumber.replace(/\s/g, '')
+            const token = await fetchToken(trimmedPhoneNumber, password);
+            const profileData = await fetchProfile(token);
+            
+            const name = `${profileData.first_name} ${profileData.last_name}` ; 
+            const mailFrom = profileData.email;
+            const accountNumber = profileData.bank_account_number;
+            setError('');
+            setIsSuccess(true);
+            setExpanded(false);
+            updateForm(name, mailFrom, accountNumber, token);
+        }
+        catch (err: any) {
+            if (typeof(err) == 'string') {
+                setError(err)
+            } 
+            else setError("Noe gikk galt.")
+        }
+    }
+    
+return (
+    <Accordion
+        defaultExpanded={false}
+        disabled={isSuccess}
+        expanded={expanded}
+        style={{marginBottom: '2em', paddingLeft: '0.5em', paddingRight: '0.5em', backgroundColor: '#a1e6c0', borderRadius: '5px'}}
```

Comment:
Maybe a less dominant background color? 😇 

```suggestion
        style={{marginBottom: '2em', paddingLeft: '0.5em', paddingRight: '0.5em', backgroundColor: '#a1e6c07a', borderRadius: '5px'}}
```